src/VscPreviewExtension/package.json

```diff
@@ -64,5 +64,10 @@
   "devDependencies": {
     "typescript": "^1.8.5",
     "vscode": "^0.11.14"
+  },
+  "dependencies": {
+    "jquery": "^3.1.0",
+    "highlightjs": "^8.7.0",
+    "d3": "^3.2.0"
```

Comment:
better way!

---
In reply to: [80203623](https://github.com/dotnet/docfx/pull/693#discussion_r80203623) [](ancestors = 80203623)